.travis.yml

```diff
@@ -1,8 +1,34 @@
+conditions: v1
 language: python
-sudo: required
-dist: xenial
-python: "3.7"
-cache: pip
+python:
+  - "3.6"
 before_install:
 - pip install -r requirements.txt
 script: pre-commit run --all-files
+
+jobs:
+  include:
+  - python: 3.7
+    dist: xenial
+    sudo: true
+
+  - stage: &stage_deploy_name upload new version of python package to PYPI (only for tagged commits)
```

Comment:
Pro tip: you can *also* upload dists from eash push to master to the Test PyPI just to always be sure that the publishing pipeline is healthy.